src/components/Layout/LayoutStyled.js

```diff
@@ -1,7 +1,14 @@
 import styled from 'styled-components';
+import { device } from '@/constants/mediaQueries';
 
 export const LayoutStyled = styled.div`
   display: grid;
-  grid-template-columns: 120px 1fr;
+  grid-template-columns: 90px 1fr;
   background-color: var(--outlet-background-color);
+  @media ${device.tablet} {
+    grid-template-columns: 90px 1fr;
```

Comment:
I beliave this is redundant.

Reply:
It is from UI/UX we had conversation and he said that for this mediaQueries should be 90px width on SideBar component

Reply:
You already defined the same styles 3 rows above.